inst/workflow/2_metrics/kri0001.yaml

```diff
@@ -60,16 +60,8 @@ steps:
       vThreshold: vThreshold
   - name: Summarize
     output: Analysis_Summary
+    save: true
     params:
       dfFlagged: Analysis_Flagged
       nMinDenominator: nMinDenominator
-  - name: list
```

Comment:
Is there a particular reason why this list of interim objects is being removed, but retained above in the config/workflows version?